### ISSUE TITLE
It would be great if you could point Eddy to a folder other than downloads.

### ISSUE THREAD

**Issue by chrisbeardy** [donadigo/eddy#86]
It would be great if there was a `Load from Custom` option to load .deb files from a folder of choice, and it also remembers your last browsing point. I like to keep a backup of all .deb files I have directing installed. So I  think it looks a bit messy when they are all just stored in the downloads folder.

**Comment by KorbsStudio** [comment]
Good idea! As I do usually keep a folder a DEB files on an external drive.
